index.md

```diff
@@ -8,6 +8,7 @@ We maintain quick links for checking End Of Life dates for various tools and tec
 - [endoflife.date/django](/django)
 - [endoflife.date/iphone](/iphone)
 - [endoflife.date/laravel](/laravel)
+- [endoflife.date/java](/java)
```

Comment:
To keep the list alphabetically sorted, Java should be moved above Laravel.

Reply:
Thanks, fixed and repushed